packages/blocks/src/blocks/core-audio.tsx

```diff
@@ -1,9 +1,23 @@
-import React, { type JSX } from 'react';
+import React from 'react';
 import {
 	cn,
 	getClassNamesFromString,
 	getStylesFromAttributes,
 } from '@snapwp/core';
+import { BlockData } from '..';
+
+export interface CoreAudioAttributes extends Record< string, unknown > {
+	autoplay?: boolean;
+	caption?: string;
+	loop?: boolean;
+	preload?: string;
+	src?: string;
+	style?: string;
+}
+
+export interface CoreAudioProps extends Omit< BlockData, 'type' > {
```

Comment:
Omit type before exporting `BlockData`

Reply:
We can't do this, since `BlockData.type` is used here:
https://github.com/rtCamp/snapwp/blob/110c3da7de0a6c9a489ae1568863f3f974b90cf4/packages/blocks/src/block-manager/index.tsx#L68-L70

Should I create a separate type just for blocks?

Reply:
Added a BlockProps type.